gwt-material/src/main/java/gwt/material/design/gen/IconTypeGenerator.java

```diff
@@ -0,0 +1,109 @@
+package gwt.material.design.gen;
+
+/*
+ * #%L
+ * GwtMaterial
+ * %%
+ * Copyright (C) 2015 GwtMaterialDesign
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import gwt.material.design.client.constants.IconType;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Generator for the {@link IconType} class. It parses the MaterialIcons-Regular.ijmap file and creates
+ * the IconType.java file.
+ * 
+ * @author gilberto-torrezan
+ * 
+ */
+public class IconTypeGenerator {
+    
+    public static void main(String[] args) throws IOException {
+        System.out.println("Generating the IconType class...");
```

Comment:
Please avoid System.out.println :) It will be better to use GWT standard logging.